generators/client/templates/angular/src/main/webapp/app/shared/_shared-libs.module.ts

```diff
@@ -9,12 +9,13 @@ import { InfiniteScrollModule } from 'angular2-infinite-scroll';
 @NgModule({
     imports: [
         NgbModule.forRoot(),
-        NgJhipsterModule.forRoot({
+        // TODO uncomment the code below and make it works with AOT
+        NgJhipsterModule/*.forRoot({
```

Comment:
like @gmarziou suggested can you try to use the ng-jhipster lib with a very minimal non jhipster app and see if AOT works?